partiql-lang/build.gradle.kts

```diff
@@ -32,12 +32,12 @@ kotlin {
 }
 
 dependencies {
-    api(project(":lib:isl"))
     api(project(":partiql-ast"))
     api(project(":partiql-spi"))
     api(project(":partiql-types"))
     api(Deps.ionElement)
     api(Deps.ionJava)
+    api(Deps.ionSchema)
```

Comment:
Where are we using this other than test? If it is just in test, can we do `testImplementation`?

Reply:
It has to do with the `partiqlisl` package -- specifically the `ResourceAuthority` that reads in isl files. It's currently being used by others.